daos-cortx/docs/setup_robinhood_policy_engine.md

```diff
@@ -0,0 +1,149 @@
+
+* Query database
+ 
+ `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/rbh-report --fs-info --class-info`
+ 
+# Test for daos to CORTX and vice-versa data movement
+
+In this exercise object movements from daos to cortx and cortx to daos will be made sure. This exercise will be carried out by defining policies inside a container and running those policies using robinhood policy engine. There are two prerequisites to carry out this test mentioned below.
+
+* Daos server node running on one VM
+
+  - follow this [document](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/docs/setup_daos.md) to setup daos and creating container.
+  
+* Cortx server node running on second VM
+
+  - follow this [document](https://github.com/Seagate/cortx/blob/main/QUICK_START.md) to setup cortx on your vm.
+  
+* Once everything is in-place let's start with the data movement test by following the steps below on daos node which is hosting robinhood policy engine.
+ 
+1. Create objects in a daos container (i.e. inside dfuse mount point)
+
+- create larger and smaller objects inside the container. container is mounted at /mnt/daos_container for this exercise and same fs_path will be used in config file as well.
+
+- These object(s) are going to be moved to s3 bucket and downloaded from there to doas container directory.
+
+2. Create config file and add policies and start Scanning the database and check contents of the container on robinhood's database
+
+- Readily available config files with correct config options are present [here.](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/src/samples/rh_daos_cortx.conf)
+
+`/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/rh_daos_cortx.conf`
+
+`/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/rbh-report --fs-info --class-info`
+ 
+3. Create s3 bucket 
+ 
+`aws s3 mb s3://daos-bucket`
+
+Note : For this exercise test bucket name used is daos-bucket and same is used inside rh_daos_cortx.conf file referenced above.
+
+4. Run the policy to move larger objects (daos_to_cortx_archive policy)
+
+`/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/rh_daos_cortx.conf --run=daos_to_cortx_archive(all)`
+
+5. Verify contents on container and on s3 bucket
+
+`ls`
+
+`aws s3 ls s3://daos-bucket`
+
+This is how we have successfully moved larger objects (size > 1MB) from daos to cortx
+
+6. Run the policy to move objects from cortx to daos(run cortx_to_daos_restore policy)
+
+`/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/rh_daos_cortx.conf --run=cortx_to_daos_restore(all)`
+
+7. Verify contents on container and on s3 bucket
```

Comment:
Put a reference for destination container

Reply:
done